app/controllers/sufia/homepage_controller_override.rb

```diff
@@ -13,10 +14,42 @@ def search_builder_class
     SearchBuilder::HomePage
   end
 
+
   protected
 
   def public_works_count
     @public_works_count ||= search_results({}).first.total
   end
   helper_method :public_works_count
+
+  def recent_items
+    how_many_works_to_show = 5
+    how_often_to_change = 60 * 10 # ten minutes in seconds
+    how_many_works_in_bag = 15
+
+    #First, put a bunch of eligible works into a bag.
+    # Note that the call to SOLR is memoized.
+    conditions =  {'read_access_group_ssim'=>'public'}
+    sort_by = ["system_modified_dtsi desc"]
+    opts = {:rows=>how_many_works_in_bag, :sort=>sort_by}
+    @works_to_pick_from ||= GenericWork.search_with_conditions( conditions, opts)
```

Comment:
since this iVar is on the whole controller, call it something that makes sense out of context. perhaps `@recent_items_to_pick_from`, or even just `@recent_items`. 

But also, instance variables on a controller are only good for the duration of a request, since a new Controller instance gets created on every request. So this won't actually cache the value between requests, if that's what you were trying to do. 

You need to store it in some kind of "global" state, such as a class variable or rails `class_attribute`.